.github/workflows/release-github.yml

```diff
@@ -31,9 +32,21 @@ jobs:
         with:
           go-version: '1.20.5'
       - name: run goreleaser
-        uses: goreleaser/goreleaser-action@v2
+        uses: goreleaser/goreleaser-action@v4
         with:
           version: latest
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract version
+        id: extract-version
+        run: |
+          printf "::set-output name=%s::%s\n" tag-name "${GITHUB_REF#refs/tags/}"
```

Comment:
We need to skip non stable versions like RC versions as homebrew only accept stable versions.